From 109d44158c4a564047790b27296d9fea0c8f5791 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 9 Aug 2023 12:16:21 +0200
Subject: [PATCH 0539/1141] dt-bindings: interrupt-controller: stm32-exti: add
 #address-cells

make dtbs_check complains for '#address-cells' property in the DT
node that does not match any of the properties listed in the
bindings.

Add the property #address-cells.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Iea040bc2f1e1bb4040ba01edfd83bc01da1182f6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/329519
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../devicetree/bindings/interrupt-controller/st,stm32-exti.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/interrupt-controller/st,stm32-exti.yaml b/Documentation/devicetree/bindings/interrupt-controller/st,stm32-exti.yaml
index 93a0cf78741a..2eff63eb3059 100644
--- a/Documentation/devicetree/bindings/interrupt-controller/st,stm32-exti.yaml
+++ b/Documentation/devicetree/bindings/interrupt-controller/st,stm32-exti.yaml
@@ -25,6 +25,8 @@ properties:
 
   "#interrupt-cells":
     const: 2
+  "#address-cells":
+    const: 0
 
   reg:
     maxItems: 1
-- 
2.39.2

